wiki/People/The_Team/fr.md

```diff
@@ -22,23 +20,23 @@ Les personnes énumérées ci-dessous constituent le noyau de l'**équipe d'osu!
 | ::{ flag=AU }:: **[peppy](https://osu.ppy.sh/users/2)** | **Chef de projet** |
 | ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Conseiller sur le modding et le mapping |
 | ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | Organisateur et administrateur de la World Cup |
-| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | Modération et conseiller communautaire |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Organisateur et administrateur de la World Cup |
 | ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Mainteneur de projet et de communauté, Sensibilisation des Featured Artist |
 | ::{ flag=JP }:: [flyte](https://osu.ppy.sh/users/3103765) | Responsable du design, Observateur résident de Creative Cloud |
-| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) | Conseiller sur le modding et le mapping |
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Organisateur et administrateur de la World Cup |
+| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Soutien aux tournois, sensibilisation des Featured Artist |
 | ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) | Développeur osu!web |
 | ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Gouvernant général, conseiller en modération |
+| ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Modération et conseiller communautaire |
 | ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | Développeur osu!web, [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
 | ::{ flag=JP }:: [notbakaneko](https://osu.ppy.sh/users/10751776) | Développeur osu!web |
-| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | Conseiller sur le modding et le mapping |
 | ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | Conseiller sur le modding et le mapping, Organisateur de la Mappers' Guild, Sensibilisation des Featured Artist |
 | ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | Gouvernant général, mainteneur du wiki |
 | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | Gouvernant général |
 | ::{ flag=AU }:: [smoogipoo](https://osu.ppy.sh/users/1040328) | Développeur d'osu!, amateur d'osu!mania, chasseur de bugs |
-| ::{ flag=US }:: [Toy](https://osu.ppy.sh/users/2757689) | Responsable du Project Loved, conseiller communautaire, Sensibilisation des Featured Artist |
 | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | Gouvernant général, assistant de tournoi |
+| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | Modération et conseiller communautaire |
+| ::{ flag=CA }:: [VINXIS](https://osu.ppy.sh/users/4323406) | Modération et conseiller communautaire |
```

Comment:
please check ALL roles with the original page https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team/en.md: VINXIS' and Naxess' roless should be mapping/modding scene advisor, not moderation: https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team/en.md